module/zfs/dbuf.c

```diff
@@ -48,6 +48,96 @@
 #include <sys/callb.h>
 #include <sys/abd.h>
 
+kstat_t *dbuf_ksp;
+
+typedef struct dbuf_stats {
+	/*
+	 * Various statistics about the size of the dbuf cache.
+	 */ 
+	kstat_named_t cache_count;
```

Comment:
Upon further reflection I don't think tracking the number of dbufs is actually all that useful.  Why don't we just keep the size in bytes and remove `cache_count` entirely.

Reply:
I chose to keep it around.